### ISSUE TITLE
Include the other colorschemes from the website

### ISSUE THREAD

**Issue by YJH16120** [arcticicestudio/nord-vim#258]
There are themes from the nord website in this [page](https://www.nordtheme.com/docs/colors-and-palettes). And I prefer the alternatives than the default theme for nord that is just different shades of blue.

**Comment by arcticicestudio** [comment]
Hi @YJH16120 👋🏼

I guess this is a wrong conclusion from reading the documentations.

Nord consists of 16 colors **in total**. These are divided into 4 palettes because they serve different purposes and uses cases when used for syntax highlighting or UI component design.
It's not like there are 16 different color themes. **Nord is a single theme that is made out of all these colors**. For example the colors from the _Polar Night_ palette are mostly used as background color shades for UI components while colors from the _Frost_ palette are used as main accent color for primary UI elements or to highlight functions/methods in code editors.

In short: **all colors from all palettes make up Nord**.

**Comment by YJH16120** [comment]
I see, so I just have to modify the hex color codes then?

**Comment by arcticicestudio** [comment]
You can simply make the theme fit your preferences by [customizing the colors and highlighting groups][1]. This allows you to change the theme without the need to fork and modify the actual source code, preventing breakage when updating to later versions.

[1]: https://www.nordtheme.com/docs/ports/vim/customization

**Comment by YJH16120** [comment]
I see. Thank you very much.
